src/test/unit/types/SdrTest.cpp

```diff
@@ -662,15 +666,17 @@ TEST(SdrTest, TestSaveLoad) {
         infile.seekg( 0 ); // rewind to start of file.
     }
 
+    cereal::JSONInputArchive json_in(infile);
+
     SDR zero_2;
-    zero_2.load( infile );
+    zero_2.load_ar( json_in );
     SDR dense_2;
-    dense_2.load( infile );
+    dense_2.load_ar( json_in );
     SDR sparse_2;
-    sparse_2.load( infile );
+    sparse_2.load_ar( json_in );
```

Comment:
please add a test for serialization with another format (binary), just a single simple SDR would be enough.

Reply:
> please add a test for serialization with another format (binary)

That would not be testing the save_ar( ) and load_ar( ) that would be a test of Cereal but ok.

Reply:
> that would be a test of Cereal but  

you're right, ok to leave it be

Reply:
@breznak you always have such good reviews.  Thanks.